backend/tenantfirstaid/session.py

```diff
@@ -25,8 +28,47 @@ def __init__(self):
         except Exception as e:
             print(e)
 
-    def get(self, session_id):
-        return json.loads(self.db_con.get(session_id) or "[]")
+    def get(self):
+        session_id = session.get("session_id")
+        if not session_id:
+            return self.getNewSessionData()
+
+        saved_session = self.db_con.get(session_id)
+        if not saved_session:
+            return self.getNewSessionData()
+
+        return json.loads(saved_session)
 
     def set(self, session_id, value):
         self.db_con.set(session_id, json.dumps(value))
+
+    def getNewSessionData(self):
+        return {
+            "city": "",
+            "state": "",
+            "messages": [],
+        }
+
+
+class InitSessionView(View):
+    def __init__(self, session: TenantSession):
+        self.session = session
+
+    def dispatch_request(self):
+        data = request.json
+        session_id = session.get("session_id")
```

Comment:
```suggestion
        session_id = session.get()
```
`TenantSession.get()` does not take additional arguments